src/main/java/com/cburch/logisim/std/ttl/Ttl7486.java

```diff
@@ -26,12 +26,10 @@
   private static class XorGateHDLGeneratorFactory extends AbstractGateHDLGenerator {
 
     @Override
-    public ArrayList<String> getLogicFunction(int index) {
-      final var contents = new ArrayList<String>();
-      contents.add("   " + Hdl.assignPreamble() + "gate_" + index + "_O" + Hdl.assignOperator()
-              + "gate_" + index + "_A" + Hdl.xorOperator() + "gate_" + index + "_B;");
-      contents.add("");
-      return contents;
+    public LineBuffer getLogicFunction(int index) {
+      return LineBuffer.getHdlBuffer()
+          .pair("xor", Hdl.xorOperator())
```

Comment:
If xor is added to addHdlPairs() this one can be removed.

Reply:
Done